src/kalastatic-kss-homepage.md

```diff
@@ -0,0 +1,14 @@
+<div class="kss__header">
+
+  <h1>Styleguide</h1>
+
+  <div class="lede">
+    <div class="content">
+      <p>A web Styleguide offer a way of ensuring consistency between brand, design and code.
+      Herein we are looking documenting every component and its code on the site in one place to ensure &quot;same-pagey&quot; communications between designers, front end developers and developers.</p>
+
+      <p>The pattern portfolio expresses every component and layout structures throughout the site.
+      It articulates the atomic design structure, and is used to illustrate the project’s shared vocabulary.</p>
+    </div>
+  </div>
+</div>
```

Comment:
This still requires us to define a homepage with some expected HTML inside to keep the design nice, rather than just Markup. May be worth some documentation in the Styleguide section.